### PR TITLE
Add missing attributes to the dateutil/relativedelta stub

### LINKED ISSUE #2269: relativedelta stub is missing attributes (e.g. years, minutes, seconds, etc.)
These are all currently missing:

```
 'day',
 'days',
 'hour',
 'hours',
 'leapdays',
 'microsecond',
 'microseconds',
 'minute',
 'minutes',
 'month',
 'months',
 'second',
 'seconds',
 'weekday',
 'year',
 'years'
```

### PR DESCRIPTION
**Pull request by bluetech**
I have deduced the types from docstring & source code of the
relativedelta class.

Fixes #2269.

### CODE REVIEW

**Comment by bluetech** [comment]
Here is the documentation: https://dateutil.readthedocs.io/en/stable/relativedelta.html

The `relativedelta(dt1, dt2)` usecase is unusable without the attributes.

File: third_party/2and3/dateutil/relativedelta.pyi [hunk 1]
```diff
@@ -17,6 +19,22 @@ SU = ...  # type: weekday
 
 
 class relativedelta(object):
+    years = ...  # type: int
```
Comment (JelleZijlstra) [inline third_party/2and3/dateutil/relativedelta.pyi]: Please use variable annotations, not type comments.

**Comment by bluetech** [comment]
Updated to use variable annotations.

**Comment by JelleZijlstra** [comment]
Thanks!

**Pull request merged**